regparser/commands/tests.py

```diff
@@ -0,0 +1,51 @@
+import click
+import importlib
+import nose
+import os
+import requests_cache
+from stevedore import extension
+from stevedore.exception import NoMatches
+
+
+def get_stevedore_module_names(namespace):
+    names = []
```

Comment:
Yes; it should live elsewhere and be pulled in by the various hooks where parser uses stevedore to  bring in extensions.